lib/routes/utgd/category.ts

```diff
@@ -58,35 +59,32 @@ async function handler(ctx) {
 
     response = await got({
         method: 'get',
-        url: apiUrl,
+        url: `${apiUrl}/${data.id}/related_articles/`,
         searchParams: {
-            type: 'article.Article',
-            fields: `article_category(category_name),article_tag(tag_name),title,article_image,article_author,article_description,article_published_time`,
-            article_category: data.id,
-            order: '-article_published_time',
-            limit,
+            page: '1',
+            pagesize,
```

Comment:
![image](https://github.com/DIYgod/RSSHub/assets/11386903/30973c62-9a37-4fe5-bba6-e253b2258ae6)
The search parameter should be `page_size`.